### PR TITLE
rafs: refine doc and consts

### PR DESCRIPTION
**Pull request by jiangliu**
Syntax only changes to refine documentation and consts.

Signed-off-by: Jiang Liu <gerry@linux.alibaba.com>

### CODE REVIEW

File: rafs/src/metadata/mod.rs [hunk 1]
```diff
@@ -76,160 +78,158 @@ pub trait RafsSuperInodes {
...
+/// Callback handler for RafsInode::walk_children_inodes().
+pub type RafsInodeWalkHandler<'a> = &'a mut dyn FnMut(
+    Option<Arc<dyn RafsInode>>,
+    OsString,
+    u64,
+    u64,
+) -> Result<RafsInodeWalkAction>;
 
-/// Trait to access metadata and data for an inode.
+/// Trait to provides readonly accessors for RAFS filesystem inode.
```
Comment (liubogithub) [inline rafs/src/metadata/mod.rs:120]: nit: provide readonly accessors.

**Review by liubogithub — APPROVED** [review]
Some minor issues, otherwise looks good.

File: rafs/src/metadata/direct_v6.rs [hunk 2]
```diff
@@ -152,7 +153,7 @@ impl Drop for DirectMappingState {
     }
 }
 
-/// Directly mmapped Rafs v6 super block.
+/// Direct-mapped Rafs v5 super block.
```
[the remark this answers is not in the document]
Comment (jiangliu) [reply rafs/src/metadata/direct_v6.rs:156]: fixed

File: rafs/src/metadata/direct_v6.rs [hunk 3]
```diff
@@ -722,6 +724,123 @@ impl RafsInode for OndiskInodeWrapper {
...
+
+        if left != 0 {
+            // Handle the rest of chunks since they shares the same content length = 0.
+            for c in chunks.iter().skip(1) {
+                content_len = std::cmp::min(chunk_size, left);
+                let desc = self.make_chunk_io(c, 0, content_len, user_io);
+
+                if desc.blob.blob_index() != descs.bi_vec[0].blob.blob_index() {
+                    trace!(
+                        "Continues storge IO has {} bios offset {} io size {} {:?}",
```
[the remark this answers is not in the document]
Comment (jiangliu) [reply rafs/src/metadata/direct_v6.rs:766]: fixed

File: rafs/src/metadata/direct_v6.rs [hunk 4]
```diff
@@ -722,6 +724,123 @@ impl RafsInode for OndiskInodeWrapper {
...
+                left -= content_len;
+                if left == 0 {
+                    break;
+                }
+            }
+        }
+
+        if !descs.bi_vec.is_empty() {
+            trace!(
+                "Continues storge IO has {} bios offset {} io size {} {:?}",
```
[the remark this answers is not in the document]
Comment (jiangliu) [reply rafs/src/metadata/direct_v6.rs:787]: fixed

File: rafs/src/metadata/mod.rs (hunk 1 above)
Comment (jiangliu) [reply rafs/src/metadata/mod.rs:120]: fixed

**Pull request merged**